examples/https_attestation/client/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "https_attestation_client"
+version = "0.1.0"
+authors = ["Ivan Petrov <ivanpetrov@google.com>"]
+edition = "2018"
+license = "Apache-2.0"
+
+[dependencies]
+anyhow = "*"
+base64 = "*"
+env_logger = "*"
+http = "*"
+hyper = "*"
+hyper-rustls = { version = "*", default-features = false, features = [
+  "webpki-tokio"
+] }
+log = "*"
+oak_abi = "=0.1.0"
+oak_attestation_common = "*"
+oak_client = { version = "=0.1.0", features = ["oak-attestation"] }
+prost = "*"
+rustls = "*"
+serde_json = "*"
+structopt = "*"
+# Pinned to 0.2 because of tonic: https://github.com/hyperium/tonic/blob/master/tonic/Cargo.toml
+tokio = { version = "0.2", features = ["fs", "macros", "sync", "stream"] }
+tonic = { version = "*", features = ["tls"] }
+regex = { version = "*" }
+tokio-rustls = { version = "*", features = ["dangerous_configuration"] }
```

Comment:
Perhaps worth a comment on why we need to enable this feature?

Reply:
Done